### PR TITLE
Fix last clippy warnings.

### LINKED ISSUE #2192: Fix last clippy lints
Lints left are:

```
yvan@P3440 ~/massa (git)-[fix-clippy-lints] % cargo clippy      
warning: all variants have the same postfix: `Error`
  --> massa-network/src/error.rs:51:1
   |
51 | / pub enum HandshakeErrorType {
52 | |     HandshakeIdAlreadyExistError(String),
53 | |     HandshakeTimeoutError,
54 | |     HandshakeInterruptionError(String),
...  |
58 | |     IncompatibleVersionError,
59 | | }
   | |_^
   |
   = note: `#[warn(clippy::enum_variant_names)]` on by default
   = help: remove the postfixes and use full paths to the variants instead of glob imports
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#enum_variant_names

warning: this function has too many arguments (10/7)
   --> massa-network/src/network_worker.rs:232:5
    |
232 | /     pub fn new(
233 | |         cfg: NetworkSettings,
234 | |         private_key: PrivateKey,
235 | |         self_node_id: NodeId,
...   |
242 | |         version: Version,
243 | |     ) -> NetworkWorker {
    | |______________________^
    |
    = note: `#[warn(clippy::too_many_arguments)]` on by default
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#too_many_arguments

warning: `massa_network` (lib) generated 2 warnings
warning: all variants have the same postfix: `Error`
  --> massa-protocol-exports/src/error.rs:57:1
   |
57 | / pub enum HandshakeErrorType {
58 | |     HandshakeIdAlreadyExistError(String),
59 | |     HandshakeTimeoutError,
60 | |     HandshakeInterruptionError(String),
...  |
64 | |     IncompatibleVersionError,
65 | | }
   | |_^
   |
   = note: `#[warn(clippy::enum_variant_names)]` on by default
   = help: remove the postfixes and use full paths to the variants instead of glob imports
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#enum_variant_names

warning: `massa_protocol_exports` (lib) generated 1 warning
warning: this function has too many arguments (9/7)
   --> massa-protocol-worker/src/protocol_worker.rs:321:5
    |
321 | /     pub fn new(
322 | |         protocol_settings: &'static ProtocolSettings,
323 | |         operation_validity_periods: u64,
324 | |         max_block_gas: u64,
...   |
330 | |         controller_manager_rx: mpsc::Receiver<ProtocolManagementCommand>,
331 | |     ) -> ProtocolWorker {
    | |_______________________^
    |
    = note: `#[warn(clippy::too_many_arguments)]` on by default
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#too_many_arguments

warning: `massa_protocol_worker` (lib) generated 1 warning
warning: this function has too many arguments (9/7)
  --> massa-consensus/src/consensus_controller.rs:40:1
   |
40 | / pub async fn start_consensus_controller(
41 | |     cfg: ConsensusConfig,
42 | |     execution_command_sender: ExecutionCommandSender,
43 | |     execution_event_receiver: ExecutionEventReceiver,
...  |
56 | |     ConsensusError,
57 | | > {
   | |_^
   |
   = note: `#[warn(clippy::too_many_arguments)]` on by default
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#too_many_arguments

warning: this function has too many arguments (13/7)
   --> massa-consensus/src/consensus_worker.rs:170:5
    |
170 | /     pub async fn new(
171 | |         cfg: ConsensusConfig,
172 | |         protocol_command_sender: ProtocolCommandSender,
173 | |         protocol_event_receiver: ProtocolEventReceiver,
...   |
183 | |         staking_keys: Map<Address, (PublicKey, PrivateKey)>,
184 | |     ) -> Result<ConsensusWorker, ConsensusError> {
    | |________________________________________________^
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#too_many_arguments

warning: `massa_consensus` (lib) generated 2 warnings
warning: this function has too many arguments (9/7)
   --> massa-node/src/main.rs:222:1
    |
222 | / async fn stop(
223 | |     bootstrap_manager: Option<BootstrapManager>,
224 | |     consensus_manager: ConsensusManager,
225 | |     consensus_event_receiver: ConsensusEventReceiver,
...   |
231 | |     api_public_handle: StopHandle,
232 | | ) {
    | |__^
    |
    = note: `#[warn(clippy::too_many_arguments)]` on by default
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#too_many_arguments

warning: `massa-node` (bin "massa-node") generated 1 warning
    Finished dev [unoptimized + debuginfo] target(s) in 0.10s
```

@damip this is about code quality and may be tackle in the current big refactoring!

_Originally posted by @yvan-sraka in https://github.com/massalabs/massa/issues/2166#issuecomment-1015619439_

### PR DESCRIPTION
**Pull request by AureliaDolo**
Fix #2192

### CODE REVIEW

**Review by yvan-sraka — DISMISSED** [review]

File: massa-protocol-exports/src/error.rs [hunk 1]
```diff
@@ -55,13 +55,13 @@ pub enum ProtocolError {
...
-    HandshakeInvalidSignatureError,
-    IncompatibleVersionError,
+    HandshakeIdAlreadyExist(String),
+    HandshakeTimeout,
+    HandshakeInterruption(String),
+    HandshakeWrongMessage,
+    HandshakeKey,
+    HandshakeInvalidSignature,
+    IncompatibleVersion,
 }
```
Comment (adrien-zinger) [inline massa-protocol-exports/src/error.rs:66]: This can totally be removed. It's unused! Remove also his reference upper in ProtocolError :)

**Review by adrien-zinger — APPROVED** [review]
LGTM with comments

File: massa-protocol-exports/src/error.rs (hunk 1 above)
Comment (AureliaDolo) [reply massa-protocol-exports/src/error.rs:66]: That's weird ... I'll look into that

Comment (AureliaDolo) [reply massa-protocol-exports/src/error.rs:66]: That was duplicated during the split of protocol and network, without checking what variants were used where

**Review by adrien-zinger — APPROVED** [review]

**Comment by AureliaDolo** [comment]
bors merge

**Pull request merged**